moveit_ros/planning/planning_request_adapter_plugins/src/fix_start_state_collision.cpp

```diff
@@ -71,6 +72,14 @@ class FixStartStateCollision : public planning_request_adapter::PlanningRequestA
     else
       ROS_INFO_STREAM("Param '" << JIGGLE_PARAM_NAME << "' was set to " << jiggle_fraction_);
 
+    if (!nh.getParam(JIGGLE_PARAM_NAME, jiggle_fraction_))
+    {
+      max_jiggle_distance_squared_ = 0.01*0.01; // = 1 cm 
```

Comment:
Sorry, but please test your commits before asking for detailed feedback. The else-case misses setting max_jiggle_distance_squared_ .

Reply:
Alright alright, I was in a rush, my bad. I fixed that and the rest.